.github/workflows/linux_ci.yml

```diff
@@ -24,7 +60,7 @@ jobs:
         with:
           path: ${{ env.pythonLocation }}
           # The cache will be rebuild every day and at every change of the dependency files
-          key: linux-${{ env.date }}-${{ hashFiles('**/setup.py') }}-${{ hashFiles('**/setup.cfg') }}-${{ hashFiles('**/pyproject.toml') }}-${{ hashFiles('.github') }}
+          key: linux-${{ env.date }}-${{ hashFiles('**/setup.py') }}-${{ hashFiles('**/setup.cfg') }}-${{ hashFiles('**/pyproject.toml') }}
```

Comment:
Don't we run into the same problem we already had back then when code changes didn't take effect in CI builds?

Reply:
Ah ok, I see: we reinstall later.